packages/calypso-build/webpack/minify.js

```diff
@@ -112,8 +112,11 @@ module.exports = ( {
 	extractComments = true,
 } = {} ) => {
 	terserOptions = {
+		compress: true,
+		mangle: {
```

Comment:
Using `mangle.properties.reserved` didn't work despite the docs

https://webpack.js.org/plugins/terser-webpack-plugin/
https://swc.rs/docs/configuration/minification

The build succeeded but e2e tests had failed, which is good cause we now know that failing to properly mangle will fail the tests.

Reply:
Beware, there are two types of mangle + reserved:
- mangle names, i.e., variable names. That's on by default, we only want to reserve (exclude) some names.
- mangle properties, where even field accesses like `response.body` are mangled. That's a specialty thing, turned off by default.

Reply:
Everything makes sense now, thanks @jsnajdr !